keanu-project/src/test/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensorTest.java

```diff
@@ -134,4 +136,15 @@ public void canApplySqrt() {
         assertEquals(Math.sqrt(2.0), result.scalar(), 0.0);
     }
 
+    @Test
+    public void play() {
```

Comment:
Looks like you forgot to delete this one

Reply:
Good catch!